File: sct_source/SourceCode/API_SoCongThuong/Models/ResultsIndustrialPromotionVotingModel.cs
using System;
using System.Collections.Generic;

namespace API_SoCongThuong.Models
{
    public partial class ResultsIndustrialPromotionVotingModel
    {
        public Guid ResultsIndustrialPromotionVotingId { get; set; }
        public bool Locallity { get; set; }
        /// <summary>
        /// Chỉ tiêu
        /// </summary>
        public Guid Targets { get; set; }
        public int NumbersRegister { get; set; }
        public int NumberCertified { get; set; }
        /// <summary>
        /// Đơn vị tính
        /// </summary>
        public string? Unit { get; set; }
        /// <summary>
        /// 1: Hoạt động 0: Không hoạt động
        /// </summary>
        public bool? IsAction { get; set; }
        /// <summary>
        /// 1: Đã xóa; 0: Chưa xóa
        /// </summary>
        public bool IsDel { get; set; }
        /// <summary>
        /// Thời gian tạo
        /// </summary>
        public DateTime CreateTime { get; set; }
        public string ActionName { get; set; } = "";
        /// <summary>
        /// Người tạo
        /// </summary>
        public Guid CreateUserId { get; set; }
        public DateTime? UpdateTime { get; set; }
        public DateTime DateRelease { get; set; }
        public Guid? UpdateUserId { get; set; }
        public string? TargetsName { get; set; } = "";
    }
}

File: sct_source/SourceCode/FE_SoCongThuong/Angular/src/app/modules/app-pages/industrial-promotion-voting-page/_models/results-industrial-promotion-voting.model.ts
import { BaseModel } from '../../../../_metronic/shared/crud-table/models/base.model';

export interface ResultsIndustrialPromotionVotingModel extends BaseModel {

  resultsIndustrialPromotionVotingId: string,
  locallity: boolean|null; //Địa bàn
  numbersRegister: number | null, //Số lượng sản phẩm đăng ký
  numberCertified: number | null, //Số lượng sản phẩm được cấp giấy chứng nhận
  targets: string,  //Số sản phẩm đăng ký tham gia bình chọn
  unit: string,
}
